modules/web-demo/src/components/EcdsaChallenge/index.tsx

```diff
@@ -12,8 +12,10 @@ const EcdsaChallenge = () => {
 
   const generateChallenge = async () => {
     setLoading(true);
+    const { loadWebAssembly } = await import('@bitgo/sdk-opensslbytes');
+    const openSSLBytes = loadWebAssembly().buffer;
     const start = new Date().getTime() / 1000;
-    const challenge = await EcdsaRangeProof.generateNtilde(3072);
```

Comment:
Just checking, where you able to test web-demo manually and verify this still works?

Reply:
I actually tested this with bitgo-ui & verified it worked there. Will update web-demo though to make sure this stays in sync